doc/classes/Node.xml

```diff
@@ -295,6 +295,11 @@
 				[b]Note:[/b] As this method walks upwards in the scene tree, it can be slow in large, deeply nested nodes. Consider storing a reference to the found node in a variable. Alternatively, use [method get_node] with unique names (see [member unique_name_in_owner]).
 			</description>
 		</method>
+		<method name="force_parent_owned">
+			<return type="void" />
+			<description>
+			</description>
```

Comment:
Needs a description, as it's quite a cryptic method on its own

Reply:
Since the method is not exposed yet, we could give it a better name.